docs/reference/mapping/fields/synthetic-source.asciidoc

```diff
@@ -28,45 +28,24 @@ PUT idx
 
 While this on the fly reconstruction is *generally* slower than saving the source
 documents verbatim and loading them at query time, it saves a lot of storage
-space.
+space. Additional latency can be avoided by not loading `_source` field in queries when it is not needed.
+
+[[synthetic-source-fields]]
+===== Supported fields
+Synthetic `_source` is supported by all field types. Depending on implementation details, field types have different properties when used with synthetic `_source`.
+
+<<synthetic-source-fields-native-list, Most field types>> construct synthetic `_source` using existing data, most commonly <<doc-values,`doc_values`>> and <<stored-fields, stored fields>>, meaning no additional space is used to store contents of `_source` field. Generated `_source` field in this case undergoes <<synthetic-source-modifications, modifications>> compared to original document caused by the storage layout of <<doc-values,`doc_values`>>.
+
+For all other fields, original value of the field is stored as is in the same way as the `_source` field in non-synthetic mode. In this case there are no modifications and field data in `_source` is the same as in original document. Similarly, malformed values of fields that use <<ignore-malformed,`ignore_malformed`>> or <<ignore-above,`ignore_above`>> need to be stored as is. This approach is less storage efficient since data needed for `_source` reconstruction is stored in addition to other data required to index the field (like `doc_values`).
```

Comment:
Nit:
s/original value of the field/the original value of a field/
s/as is in the same way/as is, in the same way/

Reply:
Appreciate these!